PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/tree/TreeManager.java

```diff
@@ -88,6 +91,38 @@ public void initialize(URL location, ResourceBundle resources) {
   }
 
   /**
+   * Set the selection manager and initialize listeners to the selection manager.
+   *
+   * @param selectionManager the selection manager.
+   */
+  private void setSelectionManager(SelectionManager selectionManager) {
+    this.selectionManager = selectionManager;
+    selectionManager.getTopGraphNode().addListener(invalid -> {
+      colorSelectedGraphNodes();
+    });
+    selectionManager.getBottomGraphNode().addListener(invalid -> {
+      colorSelectedGraphNodes();
```

Comment:
The `colorSelectedGraphNodes()` method requires no arguments, so I can't pass an invalidation listener as argument.
invalid is an InvalidationListener (which is triggered when the property becomes invalid). I don't need the actual parameters which are given to the invalidate call, so I wrote it like this.